api-test/internal/report/html.go

```
@@ -0,0 +1,1260 @@
+		})
+	}
+	sidecar := map[string]any{
+		"config":       json.RawMessage(safeJSON(o.ConfigJSON)),
+		"plan_configs": planConfigs,
+		"modules":      results,
+	}
+	if len(o.PlanConfigs) == 1 {
+		sidecar["plan_config"] = json.RawMessage(safeJSON(o.PlanConfigs[0].JSON))
+	}
+	if data, err := json.MarshalIndent(sidecar, "", "  "); err == nil {
+		_ = os.WriteFile(filepath.Join(dir, base+".json"), data, 0o644)
```

golangci-lint (2.12.2)

[medium] G306: Expect WriteFile permissions to be 0600 or less

(gosec)